### ISSUE TITLE
collapse-preserve-inline restricts users to collapse brace_style

### ISSUE THREAD

**Issue by Coburn37** [beautify-web/js-beautify#1057]
# Description
Would like an option to use any sort of brace_style while still being able to preserve functions and other statements (like `import`) that have block statements in one line. This works already with `collapse-preserve-inline` but unfortunately that means you must use the `collapse` brace_style

# Input
The code looked like this before beautification:
`w/ brace_style expand`
```
import { asdf } from "asdf";
function mine(asdf) { console.log("wowe"); }
function yours(asdf) {
    console.log("wowd");
}
```

# Expected Output
The code should have looked like this after beautification:
`w/ brace_style expand`
```
import { asdf } from "asdf";
function mine(asdf) { console.log("wowe"); }
function yours(asdf)
{
    console.log("wowd");
}
```
(should stay the same)

# Actual Output
The  code actually looked like this after beautification:
`w/ brace_style expand`
```
import
{
    asdf
}
from "asdf";
function mine(asdf)
{
    console.log("wowe");
}
function yours(asdf)
{
    console.log("wowd");
}
```

# Proposal
Add a new property called `brace_preserve_inline` which is a boolean that replaces brace_style `collapse-preserve-inline` and performs the "preserve-inline" portion of `collapse-preserve-inline` while being able to be used with any of the other `brace_style` options.

**Comment by Coburn37** [comment]
I created a PR with added tests (there's 3 test cases over 4 matrix options so 12 tests?) in #1058

**Comment by bitwiseman** [comment]
Fixed by #1052
